pkg/resourcekeeper/dispatch.go

```diff
@@ -18,17 +18,21 @@ package resourcekeeper
 
 import (
 	"context"
+	"sync"
 
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
 
-	"github.com/oam-dev/kubevela/apis/core.oam.dev/v1beta1"
 	"github.com/oam-dev/kubevela/pkg/multicluster"
 	"github.com/oam-dev/kubevela/pkg/oam"
 	"github.com/oam-dev/kubevela/pkg/resourcetracker"
 	"github.com/oam-dev/kubevela/pkg/utils/apply"
 )
 
+// MaxDispatchConcurrent is the max dispatch concurrent number
+var MaxDispatchConcurrent = 10
```

Comment:
why not const?

Reply:
MaxDispatchConcurrent is expected to be exposed to the user as a configurable parameter